Arrangement_on_surface_2/test/Arrangement_on_surface_2/cgal_test.cmake

```
@@ -1424,3 +1424,9 @@ compile_and_run(test_io)
 compile_and_run(test_sgm)
 
 compile_and_run(test_polycurve_intersection)
+if(CGAL_DISABLE_GMP)
+  get_directory_property(LIST_OF_TESTS TESTS)
+  foreach(_test ${LIST_OF_TESTS})
+    set_tests_properties(${_test} PROPERTIES ENVIRONMENT CGAL_DISABLE_GMP=1)
```

Use `APPEND` to avoid overwriting any other env variable.
```suggestion
    set_property(TEST ${_test} APPEND PROPERTY ENVIRONMENT CGAL_DISABLE_GMP=1)
```